README.md

```diff
@@ -251,6 +251,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [realtime-todo-app](https://github.com/Angarsk8/realtime-todo-app) - Realtime Todo application developed with Kemal, Websockets, React, ES2015 and PostgreSQL
  * [rocky](https://github.com/codingphasedotcom/rocky) - React Over Crystal Kemal and Yarn
  * [try.cr](https://github.com/maiha/try.cr) - Try monad
+ * [game-of-life-implementations](https://github.com/KieranP/Game-Of-Life-Implementations) - Game of Life in Crystal and other languages
```

Comment:
Fix the [specs](https://travis-ci.org/veelenga/awesome-crystal/builds/473193991)